Mathlib/AlgebraicGeometry/EllipticCurve/VariableChange.lean

```diff
@@ -64,59 +64,66 @@ namespace VariableChange
 
 variable (C C' : VariableChange R)
 
-/-- The identity linear change of variables given by the identity matrix. -/
-def id : VariableChange R :=
+private def _id : VariableChange R :=
```

Comment:
I think you can just put these into `instGroup` since they're all relatively short.

Reply:
Thank you! Let me try.

Reply:
Seems like you can replace the first line of the proof of `mul_assoc` with `simp_rw [(. * .), Mul.mul]`!

Reply:
Which one? Now I reordered the `one_def`, `mul_def` and `inv_def` lemmas (before the Group instance), so `ext <;> simp only [mul_def, Units.val_mul] <;> ring1` works without additional unfold.

Reply:
Ah of course, that works too :)